### PR TITLE
Fundraiser discover, clean up frontend code a bit

### PR DESCRIPTION
**Pull request by hoodedice**

### CODE REVIEW

File: client/src/pages/fundraisers/FundraiserCreate.js [hunk 1]
```diff
@@ -178,7 +187,8 @@ class FundraiserCreate extends Component {
         image_url: this.state.imageUrl
       };
 
-      fetch("/fundraiser", {
+      let url = process.env.REACT_APP_SERVER_URL;
```
Comment (hoodedice) [inline client/src/pages/fundraisers/FundraiserCreate.js]: TODO: change this to api method

File: client/src/pages/fundraisers/FundraisersList.js [hunk 2]
```diff
@@ -1,13 +1,146 @@
...
+        {
+          title: "Donate to impeach Lizards",
+          donated_amount: 44132
+        }
+      ]
+    };
+  }
+
+  componentDidMount() {
+    //var token = localStorage.getItem("access_token");
```
Comment (hoodedice) [inline client/src/pages/fundraisers/FundraisersList.js:53]: TODO: Implement this

**Review by hoodedice — COMMENTED** [review]
(writing notes for myself)

File: client/src/components/ProtectedRoute.js [hunk 3]
```diff
@@ -37,7 +37,7 @@ function ProtectedRoute({ children, ...rest }) {
         ) : (
           <Redirect
             to={{
-              pathname: "/signin",
+              pathname: "signin",
```
Comment (aslamm29) [inline client/src/components/ProtectedRoute.js:40]: please check if this is the correct format for pathname without '/'

File: client/src/pages/fundraisers/FundraiserCreate.js [hunk 4]
```diff
@@ -14,7 +13,6 @@ import {
 } from "@material-ui/core";
 
 import { withStyles } from "@material-ui/core/styles";
-import Snackbar from "@material-ui/core/Snackbar";
```
Comment (aslamm29) [inline client/src/pages/fundraisers/FundraiserCreate.js:17]: Please add the snackbar validation to render where necessary, you can replicate the implementation from the signin/signup pages

File: client/src/pages/fundraisers/FundraiserUpdate.js [hunk 5]
```diff
@@ -14,7 +13,6 @@ import {
 } from "@material-ui/core";
 
 import { withStyles } from "@material-ui/core/styles";
-import Snackbar from "@material-ui/core/Snackbar";
```
Comment (aslamm29) [inline client/src/pages/fundraisers/FundraiserUpdate.js:17]: you can mimic the same validation from the create component as necessary

File: client/src/components/ProtectedRoute.js (hunk 3 above)
Comment (hoodedice) [reply client/src/components/ProtectedRoute.js:40]: Yes

**Pull request closed without merging**